package.json

```diff
@@ -3,6 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@progress/kendo-licensing": "^1.1.3",
+    "@progress/kendo-react-data-tools": "^4.3.0",
+    "@progress/kendo-react-form": "^4.3.0",
+    "@progress/kendo-react-intl": "^4.3.0",
```

Comment:
What are these packages used for?

Reply:
I had been trying out different packages for error handling but they're unused should I delete them?

Reply:
Yea I will thinking you use state for error handling. Let me know if you need help and we can set up a call @esauerbo